Zend/tests/gh9775.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+GH-9775: Enum in array_unique()
+--FILE--
+<?php
+
+enum Test: string
+{
+    case AUTHENTICATED = 'authenticated';
```

Comment:
I believe it does not matter whether it is a backed enum or not? If this is correct, please add a test case for a non-backed enum.

Reply:
Correct. I added a test for that, thanks!